src/unix/utilsunx.cpp

```diff
@@ -1124,6 +1128,19 @@ wxString wxGetCpuArchitectureName()
     return wxGetCommandOutput(wxT("uname -m"));
 }
 
+wxString wxGetNativeCpuArchitectureName()
+{
+#if defined(__DARWIN__)
+    // macOS on ARM will report an x86_64 process as translated, assume the native CPU is arm64
```

Comment:
Could it make sense to explicitly check if `wxGetCpuArchitectureName()` returns "arm64"?

Reply:
It would return `x86_64` as we are a intel process in that case. But I don't think the additional check would make much sense as this "translated" property will probably not be used in another case (at least until apple changes it CPU architecture once again).